src/containers/checkout/checkout-form/tests/checkout-form.spec.js

```diff
@@ -109,70 +109,81 @@ describe('CheckoutForm tests for: ', () => {
       </BrowserRouter>
     );
   });
-  it('first name field', () => {
-    const firstNameField = screen.getByTestId('firstName').querySelector('input');
-    fireEvent.change(firstNameField, { target: { value: 'Roman' } });
-    expect(firstNameField.value).toEqual('Roman');
+  it('first name field', async () => {
+    const firstNameField = await screen.findByTestId('firstName');
+    const input = firstNameField.querySelector('input');
+    fireEvent.change(input, { target: { value: 'Roman' } });
+    expect(input.value).toEqual('Roman');
```

Comment:
add empty lines before all 'expect' for better code readability

Reply:
Done.